spec/classes/server_spec.rb

```diff
@@ -98,18 +102,23 @@
         end
 
         context 'rsyslog::server class with SELinux enabled' do
-          let(:params) {{
-            :enable_selinux => true
-          }}
-          ###it_behaves_like 'a structured module'
+          let(:facts) {
+            facts[:selinux_current_mode] = 'enforcing'
+            facts
+          }
+          it_behaves_like 'a structured module'
           it { is_expected.to contain_class('rsyslog::server::selinux') }
+          it { is_expected.not_to create_sel_boolean('nis_enabled').with({
```

Comment:
If selinux is enforcing, shouldn't we expect nis_enabled to be persistent and on?